src/chains/ethereum/ethereum/tests/api/eth/contracts/EthCall.sol

```diff
@@ -2,17 +2,23 @@
 pragma solidity ^0.8.11;
 
 contract EthCall {
-  uint public value;
+    uint256 public value;
 
-  constructor() payable {
-    value = 5;
-  }
+    constructor() payable {
+        value = 5;
+    }
 
-  function getBaseFee() public view virtual returns (uint256) {
-    return block.basefee;
-  }
+    function getBaseFee() public view virtual returns (uint256) {
+        return block.basefee;
```

Comment:
Is the indentation here squiffy or is it just me?

Reply:
solidity is supposed to be 4 spaces. this file missed that memo, i guess.